lib/routemaster/cache.rb

```diff
@@ -47,35 +47,33 @@ def initialize(&block)
       # @!attribute status
       # @return [Integer]
       # Delegated to the `block`'s return value.
-      
+
       # @!attribute headers
       # @return [Hash]
       # Delegated to the `block`'s return value.
-      
+
       # @!attribute body
       # @return pHashie::Mash]
       # Delegated to the `block`'s return value.
-      
+
       delegate :value => :@future
       delegate %i(status headers body) => :value
     end
 
     class Response < Hashie::Mash
       # @!attribute status
       # Integer
-      
+
       # @!attribute headers
       # Hash
-      
+
       # @!attribute body
       # Hashie::Mash
     end
 
-
     def initialize(redis:nil, fetcher:nil)
       @redis   = redis || Config.cache_redis
-      @expiry  = Config.cache_expiry
-      @fetcher = fetcher || Fetcher
+      @fetcher = fetcher || Fetcher.configure {}
```

Comment:
Semantically there is not a difference. I just follow what we used on the roo_http_client. We were thinking that we are going to use fetcher as our http client now like:

```ruby
fetcher = Routemaster::Fetcher.configure do |fetcher|
  fetcher.use Middleware::HateoasResponse # Will support link traversal and method addressing
end
```

Thoughts?